docs/evm-sidechain/xrpl-evm-sidechain-faq.md

```diff
@@ -0,0 +1,104 @@
+---
+html: xrpl-evm-sidechain-faq.html
+parent: intro-to-evm-sidechain.html
+blurb: Commonly asked legal questions to consider when running an EVM compatible sidechain.
+labels:
+  - Blockchain
+---
+# XRPL EVM Sidechain FAQ
+
+### Why are Ripple and PeerSyst building an EVM sidechain?
+
+The XRPL natively lacks the full programmability option to add custom business logic. The only programmabilIty option XRPL has today is a “smart transactor” but it's very difficult to write one and any significant change that affects existing protocol needs to go through the amendment process to be approved by the XRPL community. The Hooks feature developed by XRPL Labs promises to bring native programmability to the XRPL. EVM compatible blockchains have “smart contract” functionality that enables developers to build and deploy new applications without a network vote.
+
+EVM compatibility on different blockchains:
+
+| Blockchain Ecosystem | EVM Compatibility Solution/Project |
+|----------------------|------------------------------------|
+| Ethereum             | Native                             |
+| Solana               | Neon                               |
+| Polkadot             | Moonbeam                           |
+| Cosmos               | Evmos                              |
+| Polygon              | zkEVM                              |
+| BNB Chain            | BNB Smart Chain                    |
+| Avalanche            | Avalance C-chain                   |
+| **XRPL**             | **EVM Sidechain**                  |
+
+An EVM sidechain will help the XRPL ecosystem attract EVM developers who can deploy their existing EVM applications on the EVM sidechain with minor modifications.
+
+
+### What are the main benefits of the EVM sidechain?
+
+The main benefits are:
+
+- Faster time to launch by deploying applications as smart contracts.
+- No amendment approval needed to launch applications.
+- Make minor changes to existing smart contracts on other blockchains and launch them on the EVM sidechain.
```

Comment:
- You can make minor changes to existing smart contracts on other blockchains 
- You can extend the XRPL ecosystem by adding EVM ecosystem projects and developers.

[Not parallel - you can't imply the "Someone can" on the last two items if you can't add it to the first two items. There are other ways to rewrite this that you might prefer, I'm mostly pointing out the lack of parallelism.]]